### PR TITLE
UNG-2213 Get and update PoC

### PR DESCRIPTION
**Pull request by gzhk**

### CODE REVIEW

File: src/main/scala/com/ubirch/controllers/model/TenantAdminController.scala [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.ubirch.controllers.model
+
+import com.ubirch.models.poc._
+import org.joda.time.LocalDate
+
+import java.util.UUID
+
+object TenantAdminController {
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/controllers/model/TenantAdminController.scala:8]: Can you change the object name?

**Comment by gzhk** [comment]
@Hiroki6, @lalottacontinua I removed error mappers :)

**Comment by Hiroki6** [comment]
> @Hiroki6, @lalottacontinua I removed error mappers :)

Thank you!
If we decide to put the error handling in the other place in the future, we might refer to the error mappers.

**Review by Hiroki6 — APPROVED** [review]
I will merge it tomorrow.

**Pull request merged**